composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -541,6 +547,27 @@
       end
     end
 
+    context "missing native extension logs an error" do
+      let(:dependency_name) { "robaiken/missing-ext" }
```

Comment:
Is this a real extension on packagist? Or a completely fake one? 

If real, this shouldn't be under the `robaiken` namespace... we have a dedicated namespace on packagist for test fixtures: https://packagist.org/packages/dependabot/

If this is a fake extension, then composer will throw an error about unable to find the extension on packagist... ie, a 404-related error... Which is a different error than "unable to find the extension due to compilation errors"... 

A more realistic use case would be to use a PHP extension that _is_ present on packagist, but not popular enough for us to pull in: 
* https://github.com/dependabot/dependabot-core/issues/4202
* https://packagist.org/packages/phpunit/php-code-coverage

Reply:
I have removed the old fixture file and replaced it witht he suggested one